builder/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 The Kubernetes Authors. All rights reserved
```

Comment:
`s/2017/2018`

Reply:
OK, fixing. I copied this Dockerfile from another project.